example/helloWorld/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+project(alpaka-helloworld)
+cmake_minimum_required(VERSION 3.0.1)
+
+################################################################################
+# Find alpaka
+################################################################################
+SET(ALPAKA_ROOT "${CMAKE_CURRENT_SOURCE_DIR}/../../" CACHE STRING  "The location of the alpaka library")
+LIST(APPEND CMAKE_MODULE_PATH "${ALPAKA_ROOT}")
+
+find_package("alpaka" REQUIRED)
+include_directories(SYSTEM ${alpaka_INCLUDE_DIRS})
+add_definitions(${alpaka_DEFINITIONS})
+ADD_DEFINITIONS(${alpaka_DEFINITIONS} ${ALPAKA_DEV_COMPILE_OPTIONS})
```

Comment:
You have added ${alpaka_DEFINITIONS} twice.
Moreover, you should use either upper or lower case consistently for the built-in functions.